quinn-proto/src/congestion/cubic.rs

```diff
@@ -163,11 +164,13 @@ impl Controller for Cubic {
         }
     }
 
+    #[allow(clippy::branches_sharing_code)] // False positive, see discussion inline
```

Comment:
Wait, we also have an `#[allow]` on the actual `if` expression. Is it not effective? If so, we should remove it.

Reply:
I just rebased. Seems like @Matthias247 already covered it. I'll remove all related changes since it isn't related to BBR.